cirq_superstaq/custom_gates_test.py

```diff
@@ -353,6 +356,40 @@ def test_rxy() -> None:
     cirq.testing.assert_has_diagram(circuit, "0: ───Rxy(π, 0.5π)───")
 
 
+def test_parallel_rxy() -> None:
+    qubits = cirq.LineQubit.range(2)
+
+    rot_gate = cirq_superstaq.custom_gates.ParallelRxy(len(qubits), 1.23 * np.pi, 4.56 * np.pi)
```

Comment:
nit
```suggestion
    rot_gate = cirq_superstaq.ParallelRxy(len(qubits), 1.23 * np.pi, 4.56 * np.pi)
```
(various instances above/below as well)